lib/rules/no-empty-glimmer-component-classes.js

```diff
@@ -23,12 +26,20 @@ module.exports = {
   },
 
   ERROR_MESSAGE,
+  ERROR_MESSAGE_TEMPLATE_TAG,
 
   create(context) {
     return {
       ClassDeclaration(node) {
         if (isGlimmerComponent(context, node) && !node.decorators && node.body.body.length === 0) {
           context.report({ node, message: ERROR_MESSAGE });
+        } else if (
+          isGlimmerComponent(context, node) &&
```

Comment:
Can we store `isGlimmerComponent(context, node)` in a variable above since it might be expensive to calculate?

Reply:
Fixed!